components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -509,6 +540,87 @@ export default class AdvancedSettingsDisplay extends React.Component {
             }
         }
 
+        let deactiveAccountSection;
+        if (this.props.activeSection === 'deactiveAccount') {
+            deactiveAccountSection = (
+                <SettingItemMax
+                    title={
+                        <FormattedMessage
+                            id='user.settings.advance.deactiveAccountTitle'
+                            defaultMessage='Deactivate Account'
+                        />
+                    }
+                    inputs={[
+                        <div key='formattingSetting'>
+                            <div>
+                                <br/>
+                                <FormattedMessage
+                                    id='user.settings.advance.deactiveDesc'
+                                    defaultMessage='This will deactivate your account and remove your ability to log into this team site. You will no longer receive email or mobile notifications if you have those enabled.'
+                                />
+                            </div>
+                        </div>,
+                    ]}
+                    saveButtonText={'Deactivate'}
+                    setting={'formatting'}
```

Comment:
setting should be name of the section. In this case, `deactiveAccount` (or preferably `deactivateAccount`)

Reply:
done

Reply:
I guess this is not yet updated :)